### DISCUSSION TITLE
modified ports but still stuck on 3000

### DISCUSSION THREAD

**Discussion by mrhalliwell** [karakeep-app/karakeep#559] [General]
So I have modified the files below
.env (localhost:3033)
docker-compose.yml (3033:3033)
changed the port (a total of three in the two docs) into 3033.
after docker compose up, it still shows "3000/tcp, 0.0.0.0:3033->3033/tcp, :::3033->3033/tcp "

Not sure what part I missed?

**Comment by rkraken** [comment]
Try 3033:3000 in your docker-compose.

**Comment by MohamedBassem** [comment]
You shouldn't care about changing the "internal" port of the container. Only thing you need to change is in the docker compose file change `3000:3000` to `<NEW_PORT>:3000`. That should do the trick without any other change.
